pkgs/top-level/stage.nix

```
@@ -78,6 +78,8 @@ let
           gnueabihf = lib.systems.parse.abis.musleabihf;
           gnuabin32 = lib.systems.parse.abis.muslabin32;
           gnuabi64 = lib.systems.parse.abis.muslabi64;
+          gnuabielfv2 = lib.systems.parse.abis.muslabielfv2;
+          gnuabielfv1 = throw "The ELFv1 ABI is unsupported on musl";
```

```suggestion
          gnuabielfv1 = lib.systems.parse.abis.muslabielfv2;
```

Yeah I know it looks weird.  But this function is supposed to find the "closest musl approximation of" a given glibc platform.  Otherwise `pkgsStatic.*` builds will `throw "The ELFv1..."` instead of building `powerpc64-*-muslabielfv2` binaries.